dashboard/test/factories.rb

```diff
@@ -1,4 +1,8 @@
 FactoryGirl.define do
+  factory :plc_learning_resource_task, :parent => :plc_task, :class => 'Plc::LearningResourceTask' do
```

Comment:
styleguide prefers using js style hashes (when you can, ie. when the key is a symbol)
````
factory :plc_learning_resource_task, parent: :plc_task, class: 'Plc::LearningResourceTask'
````